forte_wrapper/allennlp/allennlp_processors.py

```diff
@@ -126,33 +129,41 @@ def default_configs(cls):
             'stanford_url': MODEL2URL['stanford'],
             'srl_url': MODEL2URL['srl'],
             'universal_url': MODEL2URL['universal'],
-            'cuda_devices': [-1]
+            'cuda_devices': [-1],
+            'infer_batch_size': 0
         })
         return config
 
     def _process(self, input_pack: DataPack):
         # handle existing entries
         self._process_existing_entries(input_pack)
-        sentences = [_ for _ in input_pack.get(Sentence)]
-        inputs = [{"sentence": s.text} for s in sentences]
-        results = {k: p.predict_batch_json(inputs)
-                   for k, p in self.predictor.items()}
-        for i in range(len(sentences)):
-            result = {}
-            for key in self.predictor.keys():
-                if key == 'srl':
-                    result.update(
-                        parse_allennlp_srl_results(results[key][i]["verbs"])
-                    )
-                else:
-                    result.update(results[key][i])
-            if "tokenize" in self.configs.processors:
-                # creating new tokens and dependencies
-                tokens = self._create_tokens(input_pack, sentences[i], result)
-                if "depparse" in self.configs.processors:
-                    self._create_dependencies(input_pack, tokens, result)
-                if 'srl' in self.configs.processors:
-                    self._create_srl(input_pack, tokens, result)
+
+        batch_size = self.configs['infer_batch_size']
```

Comment:
I think we need a bit type annotation here. We haven't set up the mypy pipeline, but we should be using them later. And I think mypy might complaint about some missing typing here, such as `batches`, `inputs`, `results`

Reply:
Checked with mypy, the only thing it complains is `batches` so I've fixed it. All other variables seems to be inferred correctly but added hints for them anyways.